source/node.go

```diff
@@ -102,6 +102,11 @@ func (ns *nodeSource) Endpoints(ctx context.Context) ([]*endpoint.Endpoint, erro
 			continue
 		}
 
+		if node.Spec.Unschedulable {
+			log.Debugf("Skipping node %s because it is unschedulable", node.Name)
```

Comment:
I wonder if this should not be more prominent than debug, maybe info?

Reply:
I initially went for Debug since the other "skip" step also does it in Debug: https://github.com/kubernetes-sigs/external-dns/blob/45257fb30349580aecd91f8738cf1e6f98ad3ed4/source/node.go#L100

If Info is better, i'll go for this.

Reply:
Let's have it like this